src/components/dashboard-shortcut/dashboard-shortcut.tsx

```diff
@@ -0,0 +1,68 @@
+import { Component, Prop, h } from '@stencil/core';
+
+/**
+ * @exampleComponent limel-example-dashboard-shortcut
+ * @exampleComponent limel-example-dashboard-shortcut-styling
+ */
+
+@Component({
+    tag: 'limel-dashboard-shortcut',
+    shadow: true,
+    styleUrl: 'dashboard-shortcut.scss',
+})
+export class DashboardShortcut {
+    /**
+     * Name of icon for the shortcut.
+     */
+    @Prop()
+    public icon: string;
+
+    /**
+     * The text to show below the shortcut. Long label will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label: string;
+
+    /**
+     * Set to `true` if shortcut is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean;
+
+    /**
+     * The url that the shortcut leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string;
+
+    /**
+     * Where to load the linked URL, as the name for a browsing context:
+     * - `_self`: in the current browsing context. (Default)
+     * - `_blank`: in a new tab.
+     * - `_parent`: in the parent browsing context of the current one.
+     * If no parent, behaves as `_self`.
+     * - `_top`: the topmost browsing context (the "highest" context
+     * that's an ancestor of the current one). If no ancestors, behaves as `_self`.
+     */
+    @Prop({ reflect: true })
+    public target: '_self' | '_blank' | '_parent' | '_top' = '_self';
+
+    public render() {
+        return [
+            <a
```

Comment:
First I thought we should have  `rel="noopener"`: But maybe we don't? 
https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/noopener

Reply:
I guess we could… I didn't know about that attribute before, so I haven't yet formed much of an opinion…

Reply:
it was a security thing before, but they say this:
> Note: Setting target="_blank" on <a> elements now implicitly provides the same rel behavior as setting rel="noopener" which does not set window.opener. See [browser compatibility](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#browser_compatibility) for support status.

I don't know how "new" this is. Can we take it for granted that everyone's browsers will have this security feature? Or should we add it to be safe?

Reply:
Ah, I've read up on it on [the page MDN linked to](https://mathiasbynens.github.io/rel-noopener/).

Modern browsers have this patched already, and it already wasn't a problem on cross-domain links, only on links to the same domain. It's possible there could be some situation where a user could make some sort of attack using this, but it seems fairly unlikely at first glance at least. At the very least, I think it could only be used by a user that has some kind of access to the system already, for example if an attacker has compromised the account of a normal user, and is trying to craft an attack on an admin user to gain access to that account. But the user would need to be able to upload or generate some content that would execute a script the user has written, and we really shouldn't be allowing that anywhere. If we do, that's an XSS vulnerability already, and the lack of `rel="noopener"` would "only" be making the existing vulnerability a bit worse. _If I understand things correctly._